rasa/core/channels/botframework.py

```diff
@@ -189,6 +189,23 @@ def __init__(self, app_id: Text, app_password: Text) -> None:
         self.app_id = app_id
         self.app_password = app_password
 
+    @staticmethod
+    def process_attachments(
```

Comment:
how about something like 
```suggestion
    def add_attachments_to_metadata(
```